HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentReviewViewController.swift

```diff
@@ -290,12 +290,14 @@ public final class PaymentReviewViewController: UIViewController, UIGestureRecog
         }
         let textColorString = String.rgbaHexFrom(rgbHex: paymentProvider.colors.text)
         if let textHexColor = UIColor(hex: textColorString) {
-            payButton.tintColor = UIColor.from(giniColor: GiniColor(lightModeColor: textHexColor, darkModeColor: textHexColor))
+            payButton.textColor = UIColor.from(giniColor: GiniColor(lightModeColor: textHexColor, darkModeColor: textHexColor))
         }
     }
     fileprivate func configurePayButtonInitialState() {
         payButton.disabledBackgroundColor = UIColor.from(giniColor: giniHealthConfiguration.payButtonDisabledBackgroundColor)
         payButton.isEnabled = false
+        payButton.disabledTextColor = UIColor.from(giniColor: giniHealthConfiguration.payButtonDisabledTextColor)
+        payButton.textColor = UIColor.from(giniColor: giniHealthConfiguration.payButtonTextColor)
```

Comment:
Do we need `payButtonTextColor`? It will be overridden by the `PaymentProvider`.

Reply:
Yes, the property `payButtonTextColor` will not be needed. Only if there's no payment provider as a default option.